scripts/downloadTemplate.gradle

```
@@ -30,15 +37,15 @@ task downloadTemplate(
             template = templates.keySet()[(ant.template as Integer)-1]
         }
         println "${color 'green', 'Install '+template+' documentation template.'}"
-        def targetDir = new File(new File(docDir, inputPath), 'arc42')
+        def targetDir = new File(new File(docDir, inputPath), template)
         targetDir.mkdirs()
-        if (template=='arc42') {
-            println "For more information about arc42 see https://arc42.org"
-            /**
-             if (targetDir.exists()) {println "${targetDir.canonicalPath} already exists"
-             println "in order to re-install the template, please remove the folder first and re-run the script"
-             throw new RuntimeException("Template folder already exists")}**/
-            ant.input(message: "${color 'green', 'Which language do you want to install?'}", validargs: 'CZ,EN,DE,ES,IT,NL,UA', addproperty: 'lang')
+        if (template in ['arc42', 'req42']) {
+            if (template=='arc42') {
+                println "For more information about arc42 see https://arc42.org"
+            } else {
```

```suggestion
            } else if (template == 'req42' {
```
We could throw an exception and abort in the default `else` here, if L36-37 does not already cover that
Alternative: use a switch case, throw an exception in the `default` block - but that just a matter of personal preference i guess

---

from your comments above,  i think we can ignore my suggestion then